react/Labs/ExperimentalModal/index.jsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react'
+import MuiCozyTheme from '../../MuiCozyTheme/'
+
+import Dialog from '@material-ui/core/Dialog'
+import DialogTitle from '@material-ui/core/DialogTitle'
+import DialogActions from '@material-ui/core/DialogActions'
+
+import withMobileDialog from '@material-ui/core/withMobileDialog'
+
+import AppTitle from '../../AppTitle'
+
+export const ExperimentalModal = props => {
+  const { dismissAction, fullScreen, children, open } = props
+  return (
+    <MuiCozyTheme>
+      <Dialog open={open} onClose={dismissAction} fullScreen={fullScreen}>
+        {children}
+      </Dialog>
+    </MuiCozyTheme>
+  )
+}
+ExperimentalModal.defaultProps = {
+  open: true
+}
+export const ExperimentalModalTitle = ({ children }) => {
+  return (
+    <DialogTitle disableTypography>
+      <AppTitle>{children}</AppTitle>
+    </DialogTitle>
+  )
+}
+
+export const ExperimentalModalActions = ({ type, children }) => {
+  const className = type === 'below' ? { className: 'modal_actions_below' } : {}
```

Comment:
`type` is a bit generic, maybe `layout` instead?
And with `below` I wonder "below what?", so I would suggest `stack` or `column` instead.

Reply:
row / column, no?